ldap/servers/slapd/log.c

```diff
@@ -2245,12 +2260,16 @@ log_write_title(LOGFD fp)
     char *buildnum = config_get_buildnum();
     char buff[512];
     int bufflen = sizeof(buff);
+    int rc = 0;
 
     PR_snprintf(buff, bufflen, "\t%s B%s\n",
                 fe_cfg->versionstring ? fe_cfg->versionstring : CAPBRAND "-Directory/" DS_PACKAGE_VERSION,
                 buildnum ? buildnum : "");
     LOG_WRITE_NOW_NO_ERR(fp, buff, strlen(buff), 0);
-
+    if (rc != 0) {
```

Comment:
'rc' is not set

Reply:
> > 'rc' is not set
> 
> The LOG_WRITE macros now set "rc"

Maybe we drop the macros and just write new functions to make this cleaned and easier to read?

Reply:
> > > 'rc' is not set
> > 
> > 
> > The LOG_WRITE macros now set "rc"
> 
> Maybe we drop the macros and just write new functions to make this cleaner and easier to read?

Actually reading the macro comments I now strongly feel we should clean it up.  It's talking about how NSPR performs on Windows NT :-)  I don't think these are things we need to be concerned about anymore :-)  But it should be cleaned up in a different ticket